libbeat/tests/integration/framework.go

```diff
@@ -219,6 +251,47 @@ func (b *BeatProc) Stop() {
 	}
 }
 
+// LogMatch tests each line of the logfile to see if contains any
+// match of the provided regular expression.  It will open the log
+// file on every call, read until EOF, then close it.  LogContains
+// will be faster so use that if possible.
+func (b *BeatProc) LogMatch(match string) bool {
+	re := regexp.MustCompile(match)
```

Comment:
I don't think so, because from the go docs:

```
Helper may be called simultaneously from multiple goroutines
```

And we are seeking in a file and updating offsets, I don't think we want that to happen simultaneously.